content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -42,17 +43,29 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
   - method: **Tag Chat Thread** -> **Tag Thread** and push: **Chat Thread Tagged** -> **Thread Tagged**
   - method: **Untag Chat Thread** -> **Untag Thread** and push: **Chat Thread Untagged** -> **Thread Untagged**
   - method: **Close Thread** -> **Deactivate Chat** and push: **Thread Closed** -> **Chat Deactivated**
-  - push: **Incoming Chat Thread** -> **Incoming Chat**
-- `order` has been renamed to `sort_order` in **List Chats**, **Get Chat Threads Summary** and **List Customers**.
-- `scopes` in **Multicast** has been renamed to `recipients`.
-- There're new fields `previous_thread_id` and `next_thread_id` in the [**Thread**](/messaging/agent-chat-api/v3.2/#threads) data structure.
-- Fields `order` and `timestamp` in the [**Thread**](/messaging/agent-chat-api/v3.2/#threads) data structure were replaced with the new field, `created_at` (date & time in microseconds in UTC).
+  - push: **incoming_chat_thread** -> **incoming_chat**
+- The `order` field has been renamed to `sort_order` in **List Chats**, **Get Chat Threads Summary**, and **List Customers**.
+- The `scopes` field in **Multicast** has been renamed to `recipients`.
+- There are new fields, `previous_thread_id`, `next_thread_id`, `queue` and `queues_duration` in the [**Thread**](/messaging/agent-chat-api/v3.2/#threads) data structure.
+- There's a new field `queue` in [**Chat summary**](/messaging/agent-chat-api/v3.2/#chat-summaries), nested in `last_thread_summary`.
+- The `order` and `timestamp` field in the [**Thread**](/messaging/agent-chat-api/v3.2/#threads) data structure were replaced with the new field, `created_at` (date & time in microseconds in UTC).
 - The **Get Chat Threads** method evolved into **Get Chat** ([Web](/messaging/agent-chat-api/v3.2/#get-chat) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#get-chat)). It now allows to retrieve a chat containing a particular thread. It also changed its shape; it no longer contains the `order` field and the `chat` key.
+- **Transfer Chat** ([Web](/messaging/agent-chat-api/v3.2/#transfer-chat) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#transfer-chat)) can now be used with inactive chats.
+- There are new fields, `requester_id` and `transferred_from` in [`incoming_chat`](/messaging/agent-chat-api/v3.2/rtm-reference/#incoming_chat).
+- [`chat_transferred`](/messaging/agent-chat-api/v3.2/rtm-reference/#chat_transferred) push is no longer tied directly to **Transfer Chat**, can now be generated when the transfer is implicit e.g. the agents are reassigned due to their inactivity.
```

Comment:
```suggestion
- The [`chat_transferred`](/messaging/agent-chat-api/v3.2/rtm-reference/#chat_transferred) push is no longer tied directly to **Transfer Chat**. Now, it can be generated when the transfer is implicit, for example, agents are reassigned due to their inactivity.
```